web/api/v1/api.go

```diff
@@ -441,6 +441,9 @@ func (api *API) labelValues(r *http.Request) apiFuncResult {
 var (
 	minTime = time.Unix(math.MinInt64/1000+62135596801, 0)
 	maxTime = time.Unix(math.MaxInt64/1000-62135596801, 999999999)
+
+	minTimeFormatted = minTime.Format(time.RFC3339Nano)
```

Comment:
Probably the time must be converted to `UTC` before formatting?

I get the following values for `minTime` in local time zone and in `UTC` time zones:

```
local: -292273086-05-16T18:49:10+02:02
UTC: -292273086-05-16T16:47:06Z
```

Reply:
Good catch, I've actually added it to the minTime and maxTime instead -- same outcome though :)